scripts/tests/localsetup_tests.py

```diff
@@ -818,7 +818,7 @@ def test_start_master_default(self):
                     -E, 'setup.kibana.host=kibana:5601', -E, setup.template.settings.index.number_of_replicas=0,
                     -E, setup.template.settings.index.number_of_shards=1, -E, setup.template.settings.index.refresh_interval=1ms,
                     -E, monitoring.elasticsearch=true, -E, monitoring.enabled=true,
-                    -E, apm-server.kibana.enabled=true, -E, 'apm-server.kibana.host=kibana:5601',
+                    -E, apm-server.kibana.enabled=true, -E, apm-server.agent.config.cache.expiration=2s, -E, 'apm-server.kibana.host=kibana:5601',
```

Comment:
any reason the expiry can't be even shorter?

Reply:
im fine with 1s, since this is going to be used for opbeans and everything else I didn't want it to be too noisy